test/serverTest.js

```diff
@@ -94,7 +94,7 @@ for (const supportedVersion of mc.supportedVersions) {
     }
   }
 
-  describe('mc-server ' + version.minecraftVersion, function () {
+  describe('mc-server ' + supportedVersion+'v', function () {
```

Comment:
Lint isn't running on test/serverTest since the .gitignore has

`test/server*` instead of `test/server_*` in it https://github.com/PrismarineJS/node-minecraft-protocol/blob/improve_ci_setup/.gitignore#L3

Reply:
good catch thanks